app/src/sharedTest/java/org/oppia/android/app/settings/profile/ProfileRenameActivityTest.kt

```diff
@@ -118,6 +128,19 @@ class ProfileRenameActivityTest {
     ApplicationProvider.getApplicationContext<TestApplication>().inject(this)
   }
 
+  @Test
+  fun testProfileRenameActivity_hasCorrectActivityLabel() {
+    activityTestRule.launchActivity(
+      ProfileRenameActivity.createProfileRenameActivity(
+        context = this.context.applicationContext,
+        profileId = 1
+      )
+    )
+
+    val title = activityTestRule.activity.title
```

Comment:
Please add a comment above this like the one listed [here](https://github.com/oppia/oppia-android/blob/63a754ebfaccc5f65d12a50d9a8041dd30a9b05a/app/src/sharedTest/java/org/oppia/android/app/profile/AdminAuthActivityTest.kt#L634) to provide some context about what this check is attempting to do.

Reply:
PTAL

Reply:
@justdvnsh The comment that you added does not match with the one mentioned in the reference. Please use same comment for better clarity and consistency.